composer.json

```diff
@@ -11,35 +11,33 @@
         }
     ],
     "require": {
-        "doctrine/annotations": "^1.11",
+        "doctrine/annotations": "^1.13",
```

Comment:
This version allow to use both annotation/attribute in one class, see https://tomasvotruba.com/blog/how-to-refactor-custom-doctrine-annotations-to-attributes/